app/(main)/(campaigns)/campaigns/create.tsx

```diff
@@ -91,66 +83,174 @@ const CreateCampaign = () => {
     setStages(updatedStages);
   };
 
-  const handleNext = () => {
-    let validation;
-    if (currentStep === 1) {
-      validation = stepOneSchema.safeParse({
-        campaignName,
-        campaignObjective,
-        replySpeed,
-        reminderTiming,
-      });
-    } else if (currentStep === 2) {
-      validation = stepTwoSchema.safeParse({
-        prescript,
-        campaignPurpose,
-        actorDefinition,
-        dialogues,
-      });
-    } else if (currentStep >= 3) {
-      validation = stepThreeSchema.safeParse({ stages });
-    }
+  const handleNext = async () => {
+    try {
+      console.log("currentOrganization", currentOrganization);
+      if (!user) {
+        return Toaster.error("User not found. Please login again.");
+      }
+      if (!currentOrganization?.id || currentOrganization?.id === undefined) {
```

Comment:
Isn't both these expressions same? May be it was required as !currentOrganization || !currentOrganization.id